var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -581,6 +585,11 @@ def post_configure_fixes(self):
                 spec['nccl'].prefix.include + '"',
                 '.tf_configure.bazelrc')
 
+        if spec.satisfies('@2.0.0:2.0.1'):
+            filter_file(r'\#define RUY_DONOTUSEDIRECTLY_AVX512 1',
+                        '#define RUY_DONOTUSEDIRECTLY_AVX512 0',
+                        'tensorflow/lite/experimental/ruy/platform.h')
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
will do, this is tensorflow/tensorflow#31187